src/main/java/io/github/oliviercailloux/y2018/apartments/apartment/AddressApiException.java

```diff
@@ -0,0 +1,38 @@
+package io.github.oliviercailloux.y2018.apartments.apartment;
+
+/**
+ * This class indicates that we can retrieve a good address by the API. <br>
+ * This Exception will be thrown only if the API returns an HTTP 200 but no
+ * address was found
+ */
+public class AddressApiException extends IllegalArgumentException {
```

Comment:
Ce n’est pas un problème IllegalArgument, dans ce cas.

Reply:
Fix in 94c772882bcf7e33fb78fe7833a588f4be363c90